feat: blogs like feature implemented

sumanjeet0012 opened the pull request:
<!-- If your PR fixes an open issue, use `Closes #23` to link your PR with the issue. #23 stands for the issue number you are fixing -->

## Fixes Issue

<!-- Remove this section if not applicable -->
<!-- Example: Closes #31 -->

Like feature in blogs are implemented.

## Changes proposed

<!-- List all the proposed changes in your PR -->

Blogs like service and blogs like slice are created and blogs like is added to the redux store.

## Screenshots

<!-- Add all the screenshots which support your changes -->

## Note to reviewers

<!-- Add notes to reviewers if applicable -->

<!-- --------------- -->
<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
Correct ways to mark a box:
[x] - Correct; marked as done
[X] - Correct; marked as done

Incorrect ways to mark a box:
[ ] - Incorrect; marked as not done
[x ] - Incorrect;
[ x ] - Incorrect;
[ x] - Incorrect;
-->

## Code of Conduct

- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/Support/blob/main/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/QHBPq6xP5p) community.
Feel free to check out other cool repositories of the [Thecyberworld](https://github.com/thecyberworld).
Join the Thecyberworld GitHub Organisation by raising an [issue](https://github.com/thecyberworld/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will be sent an invitation).

ArkadiK94 commented:
@sumanjeet0012 Did you implement only back-end with Redux without actually using it in the Blog component?

sumanjeet0012 commented:
> @sumanjeet0012 Did you implement only back-end with Redux without actually using it in the Blog component?

yes, I have just implemented backend and added the function and reducer in Redux but I haven't used it anywhere..
